lib/buildpacks-ci-pipeline-update-command.rb

```diff
@@ -21,7 +21,7 @@ def run!(concourse_target_name:, pipeline_name:, config_generation_command:, pip
       set-pipeline \
       --pipeline=#{pipeline_prefix}#{pipeline_name} \
       --config=<(#{config_generation_command}) \
-      --load-vars-from=<(lpass show #{buildpacks_configuration.concourse_private_filename} --notes && lpass show #{buildpacks_configuration.deployments_buildpacks_filename} --notes && lpass show #{buildpacks_configuration.repos_private_keys_filename} --notes && lpass show #{buildpacks_configuration.git_repos_private_keys_filename} --notes && lpass show #{buildpacks_configuration.git_repos_private_keys_two_filename} --notes && lpass show #{buildpacks_configuration.bosh_release_private_keys_filename} --notes) \
+      --load-vars-from=private.yml \
```

Comment:
Oh, this is awesome. If you have the decrypted file on disk, it uses that, so you can experiment without encrypting and committing multiple times.

Fixed in c528b1d4ed230259c6086b24fcac06d2f4be7d5b